internal/sourcer/registry_sourcer.go

```diff
@@ -54,15 +54,12 @@ func (cs catalogSource) Source(ctx context.Context, po *platformv1alpha1.Platfor
 }
 
 func (s sources) GetCandidates(ctx context.Context, po *platformv1alpha1.PlatformOperator) (bundles, error) {
+	// TODO(tflannag): This doesn't account for edge case where there are zero sources.
```

Comment:
This is still relevant. I think I need to refactor this `sources` type anyways, and add a test that handles the edge case where the redhat-operators catalog is unavailable.